app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
You can create variables directly on this line for later use. Like this `command, source_file, copied_file = command.split()`